tests/unit/modules/test_transactional_update.py

```diff
@@ -0,0 +1,945 @@
+import sys
```

Comment:
Can you make this a pytest test?

Reply:
Need to move this one over to pytest.  Thanks!

Reply:
Yes, done. Thanks!